ansible/idr-s3gateway.yml

```diff
@@ -28,17 +28,41 @@
       nginx_proxy_http2: true
       nginx_proxy_force_ssl: true
       # nginx_proxy_hsts_age: 31536000
-      nginx_proxy_backends:
-        - name: s3gateway
-          location: "~ ^/{{ idr_minio_s3_gateway_bucket | default('example') }}(/.*)?$"
-          server: "http://localhost:9000"
-      # Disable buffering and allow unlimited file sizes
-      # https://serverfault.com/a/818090
-      nginx_proxy_additional_directives:
-        - client_max_body_size 0
-        - proxy_http_version 1.1
-        - proxy_request_buffering off
-        - proxy_buffering off
+      nginx_proxy_sites:
+        # idr-ftp.openmicroscopy.org: S3 read-write /idr-upload only
+        - nginx_proxy_server_name: idr-ftp.openmicroscopy.org
+          nginx_proxy_backends:
+            - name: s3gateway
+              location: "~ ^/{{ idr_minio_s3_gateway_bucket | default('example') }}(/.*)?$"
+              server: "http://localhost:9000"
+          # Disable buffering and allow unlimited file sizes
+          # https://serverfault.com/a/818090
+          nginx_proxy_additional_directives:
+            - client_max_body_size 0
```

Comment:
This is only needed for the PUTs not the GETs below?

Reply:
Correct, the body of a client GET request should be very small.